.changeset/fast-rabbits-think.md

```diff
@@ -0,0 +1,8 @@
+---
+"@hashicorp/design-system-components": minor
+---
+
+`AppFrame`
+
+- Added support for `sticky-footer` slot in the grid layout
```

Comment:
thought: not sure the CSS grid is part of the AppFrame API, but `@hasStickyFooter` should be mentioned, as it's part of the API
```suggestion
- Added `@hasStickyFooter`to control the rendering of `sticky-footer` container
```